### PR TITLE
Make timeouts in tests absolute and activate test

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: test/swap/AtomicSwapEther.js [hunk 1]
```diff
@@ -6,21 +6,30 @@ function makeSwapId() {
   return `0x${crypto.randomBytes(32).toString("hex")}`;
 }
 
+/**
+ * Creates a timeout expressed as a UNIX timestamp
+ * `seconds` seconds from now.
+ */
```
Comment (willclarktech) [inline test/swap/AtomicSwapEther.js]: This explanation seems unnecessary given the clear naming.

File: test/swap/AtomicSwapEther.js [hunk 2]
```diff
@@ -132,23 +146,24 @@ contract("Cross Chain Atomic Swap with Ether", (accounts) => {
...
-      await swap.open(swapId, accounts[0], defaultHash, defaultTimeout, { from: accounts[0], value: 50000 });
+      const timeout = makeTimeout();
+      await swap.open(swapId, accounts[0], defaultHash, timeout, { from: accounts[0], value: 50000 });
 
       try {
         await swap.expire(swapId, { from: accounts[0] });
         assert.fail("expire must not succeed");
       } catch (error) {
-        expect(error).to.match(/not yet expired/i);
+        expect(error.toString()).to.match(/not yet expired/i);
```
Comment (willclarktech) [inline test/swap/AtomicSwapEther.js]: Prefer `error.message` for a more focused comparison.

Comment (webmaster128) [reply test/swap/AtomicSwapEther.js]: The issue here is I have no clue what type is thrown and toString() works almost with everything. Ideas?

Comment (willclarktech) [reply test/swap/AtomicSwapEther.js]: Did you ever try TypeScript? :p

**Pull request merged**